### PR TITLE
cloudformation:Add support arm based machines

### PR DESCRIPTION
**Pull request by yaronkaikov**
This is a preparation for https://github.com/scylladb/scylla-pkg/issues/2616. In order to support AMI promote for arm based arch, we need to generate the cloudformation yaml file based on the relevant instance types

Adding a condition based on parameter that we will pass during the template generating process
    
Thanks to @benipeled for the providing the quick solution

### CODE REVIEW

**Comment by fruch** [comment]
Should you also revert #302  in this one ?

**Comment by yaronkaikov** [comment]
@fruch Since it's not in master (next-machine-image is broken)  i reverted this commit

**Comment by yaronkaikov** [comment]
In any case. don't merge, since i will verify it with jenkins job (locally it's working)

**Comment by yaronkaikov** [comment]
Verification:
https://jenkins.scylladb.com/job/scylla-master/job/releng-testing/job/ami/1251/

@fruch Can you please approve and merge

File: aws/cloudformation/scylla.yaml.j2 [hunk 1]
```diff
@@ -94,6 +95,21 @@ Parameters:
       - i3en.12xlarge
       - i3en.24xlarge
       - i3en.metal
+{% endif %}
+{% if 'aarch64' in arch %}
```
Comment (benipeled) [inline aws/cloudformation/scylla.yaml.j2:99]: You can short it with `{% elif 'aarch64' in arch %}`

**Review by benipeled — CHANGES_REQUESTED** [review]

**Review by benipeled — APPROVED** [review]

**Review by bentsi — APPROVED** [review]

**Pull request merged**